README.Mac.md

```diff
@@ -1,294 +1,289 @@
+## 2.0 Install PHP extension: ext-ast ^0.1.5 for phan/phan
+Type into your terminal and hit enter:
+```
+pecl install ast-0.1.5;
+```
 
-### 3.5.1 Including vhosts
-Open the ```/etc/apache2/httpd.conf``` in a text editor and locate the line containing ```Include /private/etc/apache2/extra/httpd-vhosts.conf```
+## 2.1 Install Node.js
+Navigate in your browser to [NodeJS.org](https://nodejs.org/en/) to download and run the installer.
 
-Ensure that the line is uncommented (remove the ```#``` at the beginning of the line)
 
-### 3.5.2 Adding new vhost
-Open the ```/etc/apache2/extra/httpd-vhosts.conf``` in the text editor and add the following:
+## 2.2 Install LORIS
+Paste into your terminal and hit enter:
+```
+cd ~/Development/GitHub/Loris
+make dev
+cd /tools
+./install.sh
 ```
-<VirtualHost *:80>
-	ServerAdmin webmaster@localhost
-	DocumentRoot "/Users/$username/Sites/loris/htdocs/"
-	ServerName localhost.loris
-	ErrorLog "/private/var/log/apache2/loris.com-error_log"
-	CustomLog "/private/var/log/apache2/loris.com-access_log" common
-	
-	<Directory "/Users/$username/Sites/loris/htdocs/">
-		Options Indexes FollowSymLinks
-		AllowOverride All
-		Order allow,deny
-		Allow from all
-	</Directory>
-	
-	php_value include_path .:/usr/share/php:$pathToPear:/Users/$username/Sites/loris/project/libraries:/Users/$username/Sites/loris/php/libraries:/usr/local/lib/php/Smarty
-	# Possible values include: debug, info, notice, warn, error, crit,
-	# alert, emerg.
-	LogLevel warn
-
-	ServerSignature Off
 
-	#SSLEngine Off  # change to On to enable, after updating cert paths below
-	#SSLCertificateFile /etc/apache2/ssl/loris-cert.pem
-	#SSLCertificateKeyFile /etc/apache2/ssl/loris-key.pem
-	#SSLCACertificateFile /etc/apache2/ssl/CA-cacert.pem
-</VirtualHost>
+## 2.2.1 Launch the LORIS installer in your browser
+Open a browser (we officially support Firefox & Chrome) and navigate to:
 
-<VirtualHost *:80>
-	ServerName localhost
-	DocumentRoot /Library/WebServer/Documents/
-</VirtualHost>
 ```
- * <i>$username ⇾ your username for the computer</i>
- * <i>$pathToPear ⇾ path where PEAR installs .php files (see [section 1.3.4](#include_path) part 1)</i>
-
-## 3.6 Update Database
+localhost:8080/installdb.php
+```
 
-Note that the default Loris setup assumes that Loris is running on localhost. If this
-is not the case, you'll have to manually update the URL and Host config variables in the
-ConfigSettings table by running the following SQL commands from a MySQL prompt:
+or
 
-```SQL
-UPDATE Config SET Value='$yourURL' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');
-UPDATE Config SET Value='$yourHostname' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+```
+localhost:8000/installdb.php
 ```
 
-Make sure that `$yourURL` above contains the "http://" or "https://" and `$yourHostname` does not. If your server is only being accessed from localhost, you can skip this step.
+and depending if using Apache or php -S.
```

Comment:
```suggestion
depending if using Apache or php -S.
```

Reply:
Thanks @PapillonMcGill, I moved the depending if using Apache or php -S one line above because I think it makes more sense. Also I did not notice that change and unlikely if you had not commented it out to me. Thanks!